addok/ds.py

```diff
@@ -56,7 +56,9 @@ def store_documents(docs):
     for doc in docs:
         if not doc:
             continue
-        key = keys.document_key(doc['id'])
+        if '_id' not in doc:
```

Comment:
Is it a feature that we don't want to override `_id` if already in the source data? If yes, we may unit test it :)